contracts/governance/extensions/GovernorPreventLateQuorum.sol

```diff
@@ -32,9 +30,9 @@ abstract contract GovernorPreventLateQuorum is Governor {
     event LateQuorumVoteExtensionSet(uint64 oldVoteExtension, uint64 newVoteExtension);
 
     /**
-     * @dev Initializes the vote extension parameter: the number of blocks that are required to pass since a proposal
-     * reaches quorum until its voting period ends. If necessary the voting period will be extended beyond the one set
-     * at proposal creation.
+     * @dev Initializes the vote extension parameter: the time in number of blocks or seconds (depending on the governor
+     * clock mode) that is required to pass since the moment a proposal reaches quorum until its voting period ends. If
+     * necessary the voting period will be extended beyond the one set at proposal creation.
```

Comment:
```suggestion
     * necessary the voting period will be extended beyond the one set during proposal creation.
```

No strong opinion on this, but doesn't sound completely okay to me the way it's currently worded